testfiles/SolarShadingTestGPU.idf

```diff
@@ -2211,7 +2211,7 @@
     Fraction,                !- Schedule Type Limits Name
     Through: 12/31,          !- Field 1
     For: AllDays,            !- Field 2
-    Until: 24:00,0.0;!0.2;        !- Field 3
```

Comment:
@mbadams5 Looks like this schedule has no effect with PixelCounting. Comparing the GPU file with the schedule as zero and 0.2 produces no change. The PolygonClipping file does show a difference with the schedule change.